QueAns/views.py

```diff
@@ -0,0 +1,47 @@
+from django.http import HttpResponse
+from .models import Voting,Meetings,Satisfaction
+from datetime import datetime
+
+
+def upvote(request,user_id,question_id):
+    if request.method == "POST":
+        meeting = Meetings.objects.filter(question_id=question_id)
+        if meeting.end_time < datetime.now():
```

Comment:
Should we use django timezone here vs python datetime.
Read about timezone in python :- https://howchoo.com/g/ywi5m2vkodk/working-with-datetime-objects-and-timezones-in-python

Reply:
We can use Django timezone because Django uses date-time aware objects and the meetings we created also followed the same timezone. If we use Python timezone it may cause problems because it uses tzinfo attribute to store timezone information.